Products/ZenModel/migrate/updateZenPythonSvc.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+class UpdateZenPythonSvc(Migrate.Step):
+    """
+    Add resetValue and rateOption for datapoints
+    """
+
+    version = Migrate.Version(5, 0, 0)
+
+    def cutover(self, dmd):
+
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        zenpython_services = filter(lambda s: s.name == 'zenpython', ctx.services)
+        log.info("Found %i services named 'zenpython'." % len(zenpython_services))
+
+        # change the monitoring profile
+        for svc in zenpython_services:
+            m_prof = svc.monitoringProfile
+
+            # the graph config part
+            commit = False
+            for g_conf in m_prof.graphConfigs:
+                for p in g_conf.datapoints:
+                    if p.rate == True and ( not p.rateOptions ):
+                        commit = True
+                        p.rateOptions = {
+                                "counter": True,
+                                "resetThreshold": 1048576000
```

Comment:
It is an arbitrary number that is small comparing to the wrap around anomaly but at the same time large enough that will not block a valid data value.
Any suggestion of a better number will be fine.